kyo-prelude/shared/src/main/scala/kyo/kernel/Pending.scala

```diff
@@ -38,21 +38,12 @@ object `<`:
 
         inline def andThen[B, S2](inline f: Safepoint ?=> B < S2)(
             using
-            inline ev: A => Unit,
             inline frame: Frame,
             inline flatA: Flat.Weak[A],
             inline flatB: Flat.Weak[B]
         ): B < (S & S2) =
             map(_ => f)
 
-        inline def as[B, S2](other: B < S2)(
```

Comment:
Can this be moved to `kyo-combinators`?

Would it make sense to keep this if it only supported pure values?

Reply:
I think unnecessary given that `andThen` now offers the same functionality as `as`?